tests/MyGame/Example/MonsterStorage_grpc.go

```diff
@@ -30,14 +30,14 @@ func NewMonsterStorageClient(cc *grpc.ClientConn) MonsterStorageClient {
 func (c *monsterStorageClient) Store(ctx context.Context, in *flatbuffers.Builder, 
 	opts... grpc.CallOption) (* Stat, error) {
   out := new(Stat)
-  err := grpc.Invoke(ctx, "/Example.MonsterStorage/Store", in, out, c.cc, opts...)
+  err := grpc.Invoke(ctx, "/MyGame.Example.MonsterStorage/Store", in, out, c.cc, opts...)
```

Comment:
This seems unrelated to your PR, how did this end up in here?

Reply:
I questioned it myself. I just ran the generate_code.bat from windows and those files were changed, which I too thought was odd, but didn't investigate.